WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerAutomatticTracks.m

```diff
@@ -1506,7 +1506,7 @@ + (TracksEventPair *)eventPairForStat:(WPAnalyticsStat)stat
             eventName = @"reader_tag_loaded";
             break;
         case WPAnalyticsStatReaderTagPreviewed:
-            eventName = @"reader_tag_preview";
+            eventName = @"reader_tag_previewed";
```

Comment:
Same as above.

Reply:
I'm reverting this back to `reader_tag_preview`.